air/src/aux_table/bitwise/mod.rs

```diff
@@ -44,19 +54,48 @@ const B_COL_RANGE: Range<usize> = create_range(A_COL_RANGE.end, NUM_DECOMP_BITS)
 /// The index of the column containing the aggregated output value.
 const OUTPUT_COL_IDX: usize = BITWISE_TRACE_OFFSET + BITWISE_OUTPUT_COL_IDX;
 
+// PERIODIC COLUMNS
+// ================================================================================================
+
+/// Returns the set of periodic columns required by the Bitwise co-processor.
+///
+/// The columns consist of:
+/// - k0 column, which has a repeating pattern of a single one followed by 7 zeros.
+/// - k1 column, which has a repeating pattern of a 7 ones followed by a single zero.
+pub fn get_periodic_column_values() -> Vec<Vec<Felt>> {
+    vec![BITWISE_POW2_K0_MASK.to_vec(), BITWISE_POW2_K1_MASK.to_vec()]
```

Comment:
These constant names should be updated to remove the pow2 reference.

Reply:
Done! Please see [abb346a](https://github.com/maticnetwork/miden/pull/289/commits/abb346aa50875a16ebd6eb30f793475f20faf9ff).